[2023-09-13T07:38:28.434+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-09-13T07:38:28.460+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-09-13T07:38:28.461+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-13T07:38:28.497+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-08-28 00:00:00+00:00
[2023-09-13T07:38:28.511+0000] {standard_task_runner.py:57} INFO - Started process 92 to run task
[2023-09-13T07:38:28.521+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-08-28T00:00:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp3yj8l83g']
[2023-09-13T07:38:28.530+0000] {standard_task_runner.py:85} INFO - Job 17: Subtask extract_data
[2023-09-13T07:38:28.785+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-08-28T00:00:00+00:00 [running]> on host 190733975a8f
[2023-09-13T07:38:29.096+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-28T00:00:00+00:00'
[2023-09-13T07:38:29.385+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-13T07:38:29.404+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230828T000000, start_date=20230913T073828, end_date=20230913T073829
[2023-09-13T07:38:29.458+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-13T07:38:29.477+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-15T11:31:18.157+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-09-15T11:31:18.177+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-28T00:00:00+00:00 [queued]>
[2023-09-15T11:31:18.178+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-15T11:31:18.197+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-08-28 00:00:00+00:00
[2023-09-15T11:31:18.206+0000] {standard_task_runner.py:57} INFO - Started process 358 to run task
[2023-09-15T11:31:18.226+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-08-28T00:00:00+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp2vgbii3m']
[2023-09-15T11:31:18.231+0000] {standard_task_runner.py:85} INFO - Job 20: Subtask extract_data
[2023-09-15T11:31:18.325+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-08-28T00:00:00+00:00 [running]> on host 4af8904e631c
[2023-09-15T11:31:18.528+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-08-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-28T00:00:00+00:00'
[2023-09-15T11:31:18.985+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-15T11:31:19.009+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230828T000000, start_date=20230915T113118, end_date=20230915T113119
[2023-09-15T11:31:19.203+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-15T11:31:19.365+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
